### PR TITLE
New CheatSheet : Secret Service Codename

### PR DESCRIPTION
**Pull request by ColasBroux**
**What does your Instant Answer do?**
Gives the secret service codename of United States President and vice President and their families.

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Wikipedia](https://en.wikipedia.org/wiki/Secret_Service_codename)

**Why did you choose this data source?**
There is a lot of information and references at the bottom of the page are the sources of the wikipedia article.

**Are there any other alternative (better) data sources?**
Maybe

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
I dont really know 

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
I found the idea [here](https://duck.co/duckduckhack/goodie_cheat_sheets#cheat-sheet-ideas) at the bottom of the page.

**Where did you hear about DuckDuckHack? (For first time contributors)**
On DuckDuckGo

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
cheatsheet screenshot

-----
IA Page: https://duck.co/ia/view/secret_service_codenames_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/secret-service-codename.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
+{
+    "id": "secret_service_codename",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/secret-service-codename.json]: let's change this to `secret_service_cheat_sheet`

File: share/goodie/cheat_sheets/json/secret-service-codename.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    "name": "Secret Service codename",
+    "metadata": {
+        "sourceName": "Secret Service codename",
+        "sourceUrl": "https://en.wikipedia.org/wiki/Secret_Service_codename"
+    },
+    "template_type": "keyboard",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/secret-service-codename.json]: This should really use the `"reference"` template type. It's meant for cheat sheets that have textual keys/values, that aren't demonstrating keyboard shortcuts

**Comment by moollaza** [comment]
@ColasBroux  thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

**Comment by ColasBroux** [comment]
@moollaza I did the modifications you wanted me to do

**Comment by moollaza** [comment]
@ColasBroux this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@ColasBroux congrats! It's live: https://duckduckgo.com/?q=Secret+Service+Codename+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**